plugins/audit-log-node/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@janus-idp/backstage-plugin-audit-log-node",
```

Comment:
Maybe it can be  @janus-idp/backstage-plugin-audit-log-common?

Reply:
It can't because it needs to follow the backstage naming scheme.